Core/Configuration/Data/Config.cs

```diff
@@ -33,9 +33,6 @@ public class Config
   [XmlElement("prereleaseMergeIgnoreList")]
   public IgnoreList PreReleaseMergeIgnoreList { get; set; }
 
-  [XmlElement("tagStableMergeIgnoreList")]
```

Comment:
@Frazerus & @KuJo7 what happens when the element is still part of the config file in a project? will it get ignored or fail and we need to fix it everywhere?

Reply:
Just tested with @KuJo7, it does not matter.

Reply:
As we checked , it will get ignored if it is still part of the config file